src/hooks/worlds/useOwnWorlds.ts

```diff
@@ -0,0 +1,36 @@
+import { useFirestore, useFirestoreCollectionData } from "reactfire";
+import { collection, query, where } from "firebase/firestore";
+
+import { ALWAYS_EMPTY_ARRAY, COLLECTION_WORLDS } from "settings";
+
+import { World } from "api/world";
+
+import { CONVERTER_WORLD_WITH_ID } from "utils/converters";
+import { WithId } from "utils/id";
+
+import { useUser } from "hooks/useUser";
+
+export const useOwnWorlds = (): {
+  isLoading: boolean;
+  ownWorlds: WithId<World>[];
```

Comment:
Type aliases like this make it easy one day to (re)move that `WithId<>` from `utils` into `types` as per its comment.

```suggestion
  ownWorlds: WorldWithId[];
```

Reply:
Changed in https://github.com/sparkletown/sparkle/pull/2820/commits/af825f995a7267c0da1cdbc23ea8630055576519